### ISSUE TITLE
Use process.version for node version

### ISSUE THREAD

**Issue by mearns** [parshap/check-node-version#6]
The node version, at least, can be gleaned from `process.version`. Obviously that only works if you're running with node, not sure whether or not you have a use case for otherwise.

**Comment by parshap** [comment]
The difference is the current approach would tell you the version of node available in your `$PATH` whereas using `process.version` would tell you the version of node you ran `check-node-version` with. These are probably the same 99% of the time (if not 100%), but I think there's value in keeping the semantics consistent between node and npm version checking. What do you think?

And yes, my only use case is running this with node -- I don't think it'd be applicable in anhy other context.

**Comment by mearns** [comment]
That's interesting, I hadn't thought of that. I guess what I'm not really clear on is why you'd rather know what version of node is on your path than what version of node you're currently running under.

**Comment by rasenplanscher** [comment]
I think, a scenario in which this package will be used without `nodejs` is rather unlikely and could be tackled if it ever happened.

About the dichotomy between the `node` in `PATH` versus the currently running `node`, I think the 99-100% is about right, and the remaining 0-1% is only relevant if and when the node-version is checked under such a circumstance. Again, that bridge can be crossed when reached.

Checking the node version via `process.version` would by itself be faster by a large factor, but I don't think it would be really noticable from a human perspective.

FInally, having a substantially different implementation for one case would introduce both a maintenance overhead and an implementation overhead that are not currently there, in addition to the time necessary to implement in the first place. I do not think that is justified in this case.

In summary, while the idea is valid, I think this should be closed because the only patential use-case is only theoretic at present and there is no clear benefit in the change.

**Comment by rasenplanscher** [comment]
@mearns @parshap 
What do you say, close this?

**Comment by mearns** [comment]
sure @rasenplanscher

**Comment by parshap** [comment]
If anyone has a use case for this, please comment here or open a new issue.
